Assets/MixedRealityToolkit.Extensions/LostTrackingService/Assets/SR_Triangles.shader

```diff
@@ -135,11 +135,9 @@ SubShader {
 
     FragmentInput vxOut[Geo_Max_Out_Vertices];
     int stripVxCount[Geo_Max_Out_Vertices];
```

Comment:
Also while here can you clean up the file?
Not readable spacing/indenting especially at top of file, commented lines such as //bool _Color_Map_Enable_; tags are duplicated. Also some of the defines like No_Depth_Write?

Was this created via shader graph of something?

Reply:
Can do!

Not sure the source. @railboy?